### ISSUE TITLE
pip install evtx fails

### ISSUE THREAD

**Issue by truekonrads** [omerbenamram/pyevtx-rs#8]
```
C:> pip install evtx
Collecting evtx
  Downloading evtx-0.3.0.tar.gz (1.8 kB)
    ERROR: Command errored out with exit status 1:
     command: 'c:\python38-32\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-0tgd_mdr\\evtx\\setup.py'"'"'; __file__='"'"'C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-0tgd_mdr\\evtx\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\admin\AppData\Local\Temp\pip-pip-egg-info-o1hcqjr5'
         cwd: C:\Users\admin\AppData\Local\Temp\pip-install-0tgd_mdr\evtx\
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\admin\AppData\Local\Temp\pip-install-0tgd_mdr\evtx\setup.py", line 5, in <module>
        from setuptools_rust import RustExtension
    ModuleNotFoundError: No module named 'setuptools_rust'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by omerbenamram** [comment]
Hi, this is because azure pipelines doesn't have python 3.8 on their CI machines, so I cannot reliably push 3.8 wheels for all platforms.
You can install from sources (I did publish a source release newer than ancient 0.3.0), using `pip install setuptools_rust`.

I've also specifically published a 3.8 wheel for windows from my machine, but I don't want to install python 3.8 on azure machines for now, maybe I'll move to github actions some time.

**Comment by omerbenamram** [comment]
Wheels for 3.8 should be published soon.
